xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -2168,17 +2168,23 @@ bool CDVDDemuxFFmpeg::IsVideoReady()
     }
     // Workaround for live audio-only MPEG-TS streams: If there are no elementary video streams
     // present attempt to set the start time from the first available elementary audio stream instead
-    if (!hasVideo && !m_startTime)
+    if (!hasVideo)
     {
       for (unsigned int i = 0; i < m_pFormatContext->programs[m_program]->nb_stream_indexes; i++)
       {
         int idx = m_pFormatContext->programs[m_program]->stream_index[i];
         st = m_pFormatContext->streams[idx];
         if (st->codecpar->codec_type == AVMEDIA_TYPE_AUDIO)
         {
-          m_startTime = av_rescale(st->cur_dts, st->time_base.num, st->time_base.den) - 0.000001;
-          m_seekStream = i;
-          break;
+          if (st->start_time != AV_NOPTS_VALUE)
```

Comment:
what conditions are not satisfied until start_time != AV_NOPTS_VALUE?

Reply:
st->cur_dts will be unknown and set to either AV_NOPTS_VALUE or RELATIVE_TS_BASE depending on the circumstances.  The call to av_rescale() will then come up with a ridiculously high start time value, causing the stream to ultimately fail.

The latter constant is internal to ffmpeg and I didn't think introducing it would have been a good idea, so I looked for something else that was easily accessible and found that start_time would remain at AV_NOPTS_VALUE.

It would undoubtedly be better to rework some things and remove the assumption that there will always be video, but this is such a fringe case I was trying to limit any changes to this one function and not risk breaking normal MPEG-TS demuxing.

I will remove the matching test on the VIDEO portion of the logic, I thought it looked too much like a bug to have it listed for audio but not video.  I found no cases where the check is necessary for VIDEO.